### PR TITLE
little description for server file banlist.xml

### PR DESCRIPTION
**Pull request by emir404**
i created a simple description for it, maybe it can be helpful

### CODE REVIEW

File: Server/mods/deathmatch/banlist.xml [hunk 1]
```diff
@@ -1 +1,3 @@
-<banlist />
\ No newline at end of file
+<!-- This file used to store in game bans. You can get them in game with getBans fuction or you can add bans with the addBan function. -->
```
Comment (Pirulax) [inline Server/mods/deathmatch/banlist.xml]:
```suggestion
<!-- This file is used to store in-game bans. To get a list of them, use the `getBans` API function, or `addBan` to add a new ban -->
```

**Review by emir404 — COMMENTED** [review]
💯

**Comment by botder** [comment]
Does the comment disappear when you add a ban?

**Comment by tederis** [comment]
The comments aren't preserved as was pointed out above. So the benefits of this PR are dubious.

**Pull request closed without merging**